### PR TITLE
feat: support custom image

### PR DESCRIPTION
**Pull request by mob-sakai**
#### Changes

- add customImage attribute and related tests
  - from: https://github.com/webbertakken/unity-builder/pull/151
  - Users can use `unityci/editor`, `gableroux/unity3d`, and other images with any tags.

#### Checklist

- [x] Read the contribution [guide](../CONTRIBUTING.md) and accept the [code](../CODE_OF_CONDUCT.md) of conduct
- [x] Readme (updated or not needed)
- [x] Tests (added, updated or not needed)

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -6,6 +6,10 @@ inputs:
     required: false
     default: '2019.2.11f1'
     description: 'Version of unity to use for testing the project.'
+  customImage:
+    required: false
+    default: ''
+    description: 'Specific docker image that should be used for building the project'
```
Comment (webbertakken) [inline action.yml:12]:
```suggestion
    description: 'Specific docker image that should be used for testing the project'
```

**Comment by webbertakken** [comment]
Looks good. Could you also add the parameter in [unityci/website](https://github.com/Unity-CI/Website) in the documentation section for unity actions / tests.

**Review by webbertakken — APPROVED** [review]

**Comment by mob-sakai** [comment]
Sure.

**Comment by mob-sakai** [comment]
PR: Unity-CI/Website#59

**Pull request merged**